Doc/library/stdtypes.rst

```diff
@@ -39,26 +39,17 @@ Truth Value Testing
    single: false
 
 Any object can be tested for truth value, for use in an :keyword:`if` or
-:keyword:`while` condition or as operand of the Boolean operations below. The
-following values are considered false:
+:keyword:`while` condition or as operand of the Boolean operations below.
 
   .. index:: single: None (Built-in object)
-
-* ``None``
-
   .. index:: single: False (Built-in object)
 
-* ``False``
-
-* zero of any numeric type, for example, ``0``, ``0.0``, ``0j``.
-
-* any empty sequence, for example, ``''``, ``()``, ``[]``.
-
-* any empty mapping, for example, ``{}``.
-
-* instances of user-defined classes, if the class defines a :meth:`__bool__` or
-  :meth:`__len__` method, when that method returns the integer zero or
-  :class:`bool` value ``False``. [1]_
+Objects are considered false if a :meth:`__bool__` or :meth:`__len__` method is
+defined on them, when that method returns the integer zero or :class:`bool`
+value ``False``. [1]_  Practically, this means that empty containers (such as
+``[]``, ``()``, ``{}``, ``''``, etc.) are false and that numbers equal to zero
```

Comment:
I strongly prefer 'collection' rather than 'container'.  Collections do not 'contain' python objects.  They may contain references to objects, values corresponding to objects, or info needed to generate objects by algorithm.